<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Matriculacion</class>
 <widget class="QWidget" name="Matriculacion">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>419</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>631</width>
    <height>419</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>631</width>
    <height>419</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Matriculacion</string>
  </property>
  <widget class="QLineEdit" name="lnEdit_Matricula">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>48</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdit_Nombre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>108</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdit_Apellidos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>168</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdit_Curso">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>228</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lnEdit_Ciclo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>288</y>
     <width>241</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Matricula">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>51</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Nro. Matricula:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Nombre">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>111</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Apellidos">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellidos:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Curso">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>230</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Cruso:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Ciclo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>290</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Ciclo:</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>90</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertLay_Sexo">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QRadioButton" name="rad_Masculino">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Hombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QRadioButton" name="rad_Femenino">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Mujer</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>230</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="vertLay_Situacion">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QCheckBox" name="check_Estudia">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Estudia</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="check_Trabaja">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>Trabaja</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="lbl_Sexo">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>70</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Sexo:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_Situacion">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>210</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Situaci√≥n:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Modificar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Modificar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_Borrar">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>360</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Borrar</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
